# joint_limits.yaml allows the dynamics properties specified in the URDF to be overwritten or augmented as needed
# Specific joint properties can be changed with the keys [max_position, min_position, max_velocity, max_acceleration]
# Joint limits can be turned off with [has_velocity_limits, has_acceleration_limits]
joint_limits:
  base_to_motor1:
    has_velocity_limits: true
    max_velocity: 0.314
    has_acceleration_limits: false
    max_acceleration: 0
  extension:
    has_velocity_limits: true
    max_velocity: 5
    has_acceleration_limits: false
    max_acceleration: 0
  finger1_to_gripper_motor:
    has_velocity_limits: true
    max_velocity: 0.5
    has_acceleration_limits: false
    max_acceleration: 0
  finger2_to_gripper_motor:
    has_velocity_limits: true
    max_velocity: 0.5
    has_acceleration_limits: false
    max_acceleration: 0
  swivel_to_motor2:
    has_velocity_limits: true
    max_velocity: 0.314
    has_acceleration_limits: false
    max_acceleration: 0
  wrist_motor_to_extension:
    has_velocity_limits: true
    max_velocity: 0.5
    has_acceleration_limits: false
    max_acceleration: 0